common/app/views/fragments/immersiveGarnettMainMedia.scala.html

```diff
@@ -29,7 +29,7 @@
                     <div class="badge-slot">
                         <a href="@LinkTo {/@badge.seriesTag}">
                             <img class="badge-slot__img" src="@badge.imageUrl" alt=""/>
-                            <span class="u-h">View more @Localisation(page.item.content.sectionLabelName) articles</span>
+                            <span class="u-h">View more @Localisation(page.item.content.sectionLabelName.getOrElse("")) articles</span>
```

Comment:
I suppose we shouldn't show this span at all if `sectionLabelName` is `None`

Reply:
Actually yes, but I didn't want to change the existing behaviour, but yes. I didn't give much thoughts to it, I might later if I see that view remains relevant despite DCR 100%